src/test/regress/sql/single_node.sql

```diff
@@ -1020,6 +1020,23 @@ DROP TABLE test_disabling_drop_and_truncate_102040;
 RESET citus.shard_replication_factor;
 DROP TABLE test_disabling_drop_and_truncate;
 
+-- test creating distributed or reference tables from shards
+SET citus.shard_replication_factor TO 1;
```

Comment:
minor: It would be the same if we could remove the reset command on line 1020, instead of adding a set command here

We already have shard_replication_factor set to 1, on line 1019. We reset that on the next line (1020), and here (1024) we set it back to 1 again.